sql/executor.go

```diff
@@ -39,11 +42,18 @@ var errTransactionInProgress = errors.New("there is already a transaction in pro
 type Executor struct {
 	db     client.DB
 	nodeID uint32
+
+	// System Config and mutex.
+	systemConfig   *config.SystemConfig
```

Comment:
I thought about that, but I prefer to avoid the use of `unsafe` when not absolutely necessary.
object + associated mutex should be familiar enough for everyone that locking will not be an issue.